avaframe/com1DFA/com1DFACfg.ini

```diff
@@ -109,7 +109,7 @@ tEnd = 400
 # to use a variable time step (time step depends on kernel radius)
 sphKernelRadiusTimeStepping = False
 # Upper time step limit coefficient if option sphKernelRadiusTimeStepping is chosen.
-cMax = 0.5
+cMax = 0.02
```

Comment:
Is this on purpose?

Reply:
Yes, 0.02 means a time step of 0.1s if the mesh size is 5m which is the default value we want